tests/short_term/test_summarization.py

```diff
@@ -0,0 +1,393 @@
+from typing import List
+
+from langchain_core.messages import (
+    AIMessage,
+    BaseMessage,
+    HumanMessage,
+    SystemMessage,
+)
+
+from langmem.short_term.summarization import summarize_messages
+
+
+class MockChatModel:
```

Comment:
Unless this mock chat model is doing something very special. I'd rather we used 

langchain_core.chat-Models.fake import GenericChatModel